test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+    function testFunction1() public {}
+    function testFunction2() public {}
+    function testFunction3() public {}
+
+
+    function() external [] externalArray0 = new function() external[] (3);
+    function() external [] externalArray1 = [this.testFunction1, this.testFunction2, this
+    .testFunction3];
+
+    function copyExternalStorageArrayOfFunctionType() external returns (bool)
+    {
+        externalArray0 = externalArray1;
+        return keccak256(abi.encodePacked(externalArray0)) == keccak256(abi.encodePacked(externalArray1));
```

Comment:
Might be a good idea to also compare them before assingment to make sure they were really different.
```suggestion
        assert(keccak256(abi.encodePacked(externalArray0)) != keccak256(abi.encodePacked(externalArray1)));
        externalArray0 = externalArray1;
        return keccak256(abi.encodePacked(externalArray0)) == keccak256(abi.encodePacked(externalArray1));
```